.circleci/config.yml

```diff
@@ -14,21 +17,22 @@ aliases:
 
   - &restore-yarn-cache
     keys:
-      - yarn-{{ checksum "yarn.lock" }}-{{ checksum ".circleci/config.yml" }}
+      - yarn-{{ arch }}-{{ checksum "yarn.lock" }}-{{ checksum ".circleci/config.yml" }}
 
   - &save-yarn-cache
     paths:
       - node_modules
       - ~/.npm
       - ~/.cache
-    key: yarn-{{ checksum "yarn.lock" }}-{{ checksum ".circleci/config.yml" }}
+    key: yarn-{{ arch }}-{{ checksum "yarn.lock" }}-{{ checksum ".circleci/config.yml" }}
 
 defaults: &defaults
-  working_directory: ~/buie
+  working-directory: ~/buie
   docker:
-    - image: cypress/base:14.19.0
+    - image: cypress/included:12.13.0
```

Comment:
using only one image for all since the keys are the same for all jobs, otherwise throws permission error on subsequent runs